02_configure_host.sh

```diff
@@ -198,3 +198,15 @@ sudo "${CONTAINER_RUNTIME}" run -d --net host --privileged --name vbmc ${POD_NAM
 sudo "${CONTAINER_RUNTIME}" run -d --net host --privileged --name sushy-tools ${POD_NAME_INFRA} \
      -v "$WORKING_DIR/virtualbmc/sushy-tools":/root/sushy -v "/root/.ssh":/root/ssh \
      "${SUSHY_TOOLS_IMAGE}"
+
+# Installing the openstack/ironic clients on the host is optional
+# if not installed, we use an alias to openstackclient.sh which
+# runs the clients in a container (metal3-io/ironic-client)
+if ! command -v openstack; then
+  if ! grep -q "^alias openstack" ~/.bashrc; then
+    echo "alias openstack=\"${SCRIPTDIR}/openstackclient.sh\"" >> ~/.bashrc 
+    # Set also for the current shell
+    # shellcheck disable=SC2139
+    alias openstack="${SCRIPTDIR}/openstackclient.sh"
```

Comment:
Do aliases created in a script persist? They don't seem to in my testing

Reply:
I thought it did but will check, thanks

Reply:
Yeah you're right, it makes sense that we can't set something in the parent shell from the subshell so I must have made a mistake in my testing.

I guess the only option which won't require starting a new shell session is to copy or symlink the wrapper script to somewhere already on the $PATH?

Reply:
To be able to make that persistent in the current shell, you'd have to source the script and not just run it.
Best way to create a permanent alias would be adding it to .bashrc, as you're doing, but I'm not sure it's a good idea, I'd rather add a message as @stbenjam suggested.
If we want to redirect the openstack command to the openstackclient.sh script inside the scripts, I suggest to use a function or redefine the openstack command as a variable.

Reply:
Looking at `01_prepare_host.sh` I see we do use /usr/local/bin by default for kubectl:

```
KUBECTL_PATH="${KUBECTL_PATH:-/usr/local/bin/kubectl}"
```

Perhaps I should do the same and copy the wrapper script to /usr/local/bin/openstack (with the option to override the location e.g OPENSTACK_CLIENT_PATH similar to the above?

That should work in most cases except on some platforms running as root I think?